common/djangoapps/student/views/management.py

```diff
@@ -987,10 +988,13 @@ def validate_secondary_email(account_recovery, new_email):
     if new_email == account_recovery.user.email:
         raise ValueError(_('Cannot be same as your sign in email address.'))
 
-    # Make sure that secondary email address is not same as any of the primary emails.
-    message = get_email_existence_validation_error(new_email)
```

Comment:
`get_email_existence_validation_error` appears to already use `email_exists_or_retired` under the hood, which suggests it worked before, or am I missing something?

Reply:
FYI @BessieSteinberg @tuchfarber

Reply:
@pwnage101 I wanted to use the following error in the case that the email already existed ```"It looks like {email} belongs to an existing account. Try again with a different email address."``` My understanding was that if I used ```get_email_existence_validation_error``` I couldn't customize the error message.